packages/webcrypto-ed25519-polyfill/src/__tests__/secrets-test.ts

```diff
@@ -56,31 +56,120 @@ const ED25519_PKCS8_HEADER =
     ]);
```

Comment:
<i class='graphite__hidden'>[Re: lines 26 to 26]<br/></i>

I feel like at no point do we need this as a `Uint8Array`. We only ever use the array of bytes and spread them in places.

Unwrap this maybe?<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1756?utm_source=unchanged-line-comment">Graphite</a>.</span>